owl/SOMA-HOME.owl

```diff
@@ -220,6 +235,38 @@
     
 
 
+    <!-- http://www.ease-crc.org/ont/SOMA-HOME.owl#Oven -->
+
+    <owl:Class rdf:about="http://www.ease-crc.org/ont/SOMA-HOME.owl#Oven">
+        <rdfs:subClassOf rdf:resource="http://www.ease-crc.org/ont/SOMA.owl#Appliance"/>
+        <rdfs:subClassOf rdf:resource="http://www.ease-crc.org/ont/SOMA.owl#DesignedContainer"/>
+        <rdfs:subClassOf>
+            <owl:Restriction>
+                <owl:onProperty rdf:resource="http://www.ease-crc.org/ont/SOMA.owl#hasDisposition"/>
+                <owl:someValuesFrom rdf:resource="http://www.ease-crc.org/ont/SOMA.owl#Containment"/>
+            </owl:Restriction>
+        </rdfs:subClassOf>
+        <rdfs:subClassOf>
+            <owl:Restriction>
+                <owl:onProperty rdf:resource="http://www.ease-crc.org/ont/SOMA.owl#hasDisposition"/>
```

Comment:
I don't think this is the way to go to assert the oven has a disposition to expose its contents to heat. It is not clear that a bakedgood will be a trigger only for such a disposition, nor is it true that this disposition requires a baked good to be involved in the baking process. Perhaps I want to use the oven to melt butter, for example, or age a piece of paper for a school project in history class.

If I recall, there is a Tempering disposition. It, or some appropriate subclass of it, should be used.